.github/workflows/release-beta.yml

```
@@ -0,0 +1,49 @@
+#    if: |
+#      startsWith(github.event.head_commit.message, 'chore(release):') &&
+#      contains(github.event.head_commit.message, '-beta.')
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      NPM_CONFIG_PROVENANCE: true
+      TAG: ${{ inputs.tags }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: echo
+        run: |
```

actionlint (1.7.4)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)